python/ray/_private/services.py

```diff
@@ -1605,6 +1606,8 @@ def start_raylet(
             available externally to this node.
         env_updates: Environment variable overrides.
         labels: The key-value labels of the node.
+        enable_physical_mode: Whether physical mode is enaled, which applies
```

Comment:
typo and grammar

Also it's not clear what resource consumptions are constrained. memory? cpu? gpu?